substrate/frame/staking/src/benchmarking.rs

```diff
@@ -246,50 +255,62 @@ benchmarks! {
 
 		let stash = scenario.origin_stash1.clone();
 		let controller = scenario.origin_controller1;
-		let original_bonded: BalanceOf<T>
-			= Ledger::<T>::get(&controller).map(|l| l.active).ok_or("ledger not created after")?;
+		let original_bonded: BalanceOf<T> = Ledger::<T>::get(&controller)
+			.map(|l| l.active)
+			.ok_or("ledger not created after")?;
 
 		let _ = asset::mint_existing::<T>(&stash, max_additional).unwrap();
 
 		whitelist_account!(stash);
-	}: _(RawOrigin::Signed(stash), max_additional)
-	verify {
+
+		#[extrinsic_call]
+		_(RawOrigin::Signed(stash), max_additional);
+
 		let ledger = Ledger::<T>::get(&controller).ok_or("ledger not created after")?;
 		let new_bonded: BalanceOf<T> = ledger.active;
 		assert!(original_bonded < new_bonded);
+
+		Ok(())
 	}
 
-	unbond {
+	#[benchmark]
+	fn unbond() -> Result<(), BenchmarkError> {
 		// clean up any existing state.
 		clear_validators_and_nominators::<T>();
 
 		// setup the worst case list scenario.
-		let total_issuance = asset::total_issuance::<T>();
+		let _total_issuance = asset::total_issuance::<T>();
 		// the weight the nominator will start at. The value used here is expected to be
 		// significantly higher than the first position in a list (e.g. the first bag threshold).
 		let origin_weight = BalanceOf::<T>::try_from(952_994_955_240_703u128)
 			.map_err(|_| "balance expected to be a u128")
 			.unwrap();
 		let scenario = ListScenario::<T>::new(origin_weight, false)?;
 
-		let stash = scenario.origin_stash1.clone();
+		let _stash = scenario.origin_stash1.clone();
```

Comment:
why not used?

Reply:
I think the old benchmarking code did not allow clippy and the compiler to emit unused warnings etc.  
So it was probably already unused but nobody noticed.